content/css/concepts/cursor/cursor.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'cursor'
+Description: 'Sets the appearance of the mouse cursor when hovering over an element.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Style'
+  - 'UI'
+CatalogContent:
+  - 'learn-css'
+  - 'learn-intermediate-css'
+---
+
+The `cursor` property sets the appearance of the mouse cursor when hovering over an element. The appearance can be set by specifying the type of cursor, or by loading a specific icon. When using an icon, fallback images can be used, ending with a mandatory keyword fallback.
+
+These are some common keywords used to specify the type of cursor to display:
+
+![Cursors](https://raw.githubusercontent.com/Codecademy/docs/main/media/cursors-1.png)
+
+## Syntax
```

Comment:
This is me, personally, but I would put the image + supporting text between the "Syntax" and "Example" sections.